cmd/deploy/deploy_test.go

```diff
@@ -0,0 +1,125 @@
+package deploy
```

Comment:
I misunderstood what you said yesterday about adding contributing to this branch, here is what I threw together yesterday: https://paste.googleplex.com/6488835253862400. Never uploaded the commit though :(

If there are any sections you prefer feel free to add them. The only major thing was the kind waiting I added a comment for already